ui-particles-angular/projects/ui-policy-studio-angular/src/lib/models/testing.ts

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
`testing.ts` like `index.ts` allows to export easily what should be public (testing) and used by test or storybook.

we can find another name if you want I'm not fully satisfied but I have no better simple idea 🤷‍♂️

Reply:
I would suggest `testing-barrel.ts` to understand it only deals with exports ?
or `testing/index.ts` ?

Reply:
what about `index-testing.ts` ? to understand that it is the equivalent of the index.ts 🤷‍♂️ 

already that "index" should be `public-api.ts` for what is exposed by the lib and index what is private. but i think it is not necessary to go into that level of detail

Reply:
`index-testing.ts` can be a nice compromise